develop-docs/ingestion/relay-best-practices.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Relay Best Practices
+---
+
+Relay is a critical component in Sentry's infrastructure handling hundreds of thousands of requests per second.
+To make sure your changes to Relay go smoothly, there are some best practices to follow.
+
+
+## Forward Compatibility
+
+Make sure changes to the event protocol and APIs are forward-compatible. Relay should not drop
+or truncate data it does not understand. It is a supported use-case to have customers running
+outdated Relays but up-to-date SDKs.
+
```

Comment:
We could mention `retain=true` here.

Reply:
Not sure how to phrase this and if we start naming things we probably want to name all the tools available (e.g. `additional_properties`). Gonna leave this for a follow-up or maybe even an extra page.